icu4c/source/common/ubrk.cpp

```diff
@@ -174,6 +174,10 @@ ubrk_safeClone(
     return (UBreakIterator *)newBI;
 }
 
+U_CAPI UBreakIterator * U_EXPORT2
+ubrk_clone(const UBreakIterator *bi, UErrorCode *status) {
+    return ubrk_safeClone(bi, nullptr, nullptr, status);
```

Comment:
This will return a U_SAFECLONE_ALLOCATED_WARNING from ubrk_safeClone(), which you don't want. It's probably better to just call bi->clone() directly.

Reply:
Ah. Thanks for spotting it! Done now.